mr/mr_test.cc

```diff
@@ -56,7 +62,10 @@ class StrValMapper {
 
     cntx->Write(std::move(a));
   }
+ private:
+  static std::atomic<int> should_stop_;
```

Comment:
Do you set it somewhere? Why is it static?

Reply:
See how I use it. Its only goal is to cause some calls to call `boost::this_fiber::yield`.
Schau, wie ich es benutze. Sein Ziel ist nur, zu verursachen, dass manche Aufrufe `boost::this_fiber::yield` aufrufen.

Reply:
I understand how you use it. I do not see where you set it

Reply:
I don't because I don't care about its initial value. I thought that the fact that I do not set it documents it better. Since I see that this is not the case, I'll fix.

Ich initialisiere sie nicht, weil ihre Anfangswert mir egal ist. Ich habe gedacht, dass keine Initialisierung dies besser erklärt. Weil ich sehe, dass es erklärt nicht, werde ich es beheben.